.github/workflows/publish-binaries.yml

```diff
@@ -127,10 +125,10 @@ jobs:
     runs-on: ubuntu-latest
     needs: check-version
     env:
-      ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
+      DEBIAN_FRONTEND: noninteractive
```

Comment:
https://askubuntu.com/questions/909277/avoiding-user-interaction-with-tzdata-when-installing-certbot-in-a-docker-contai

and avoid 
<img width="721" alt="Capture d’écran 2024-07-09 à 13 57 15" src="https://github.com/meilisearch/meilisearch/assets/20380692/157addd7-5fc2-427d-82e9-5097d2e80911">